lighthouse-core/report/v2/report-generator.js

```diff
@@ -99,6 +100,15 @@ class ReportGeneratorV2 {
         if (typeof result.score === 'boolean') {
           auditScore = result.score ? 100 : 0;
         }
+        // If a result was not applicable, meaning its checks did not run against anything on
+        // the page, force it's weight to 0. It will not count during the arithmeticMean() but
+        // will still be included in the final report json and displayed in the report as
+        // "Not Applicable".
+        if (result.notApplicable) {
+          audit.weight = 0;
+          result.informative = true;
```

Comment:
super nit: can we move this line to be the last of the three? just something about me expects to see score, weight, other...in that order 😆